prime-dotnet-webapi/ViewModels/Enrollee/EnrolmentCardViewModel.cs

```diff
@@ -19,16 +17,11 @@ public string RemoteAccess
         {
             get
             {
-                var enrolleeRemoteUsers = Submission.ProfileSnapshot.Value<JArray>("enrolleeRemoteUsers");
-
-                if (enrolleeRemoteUsers != null)
+                if (Submission.RequestedRemoteAccess)
                 {
-                    if (enrolleeRemoteUsers.Count > 0)
-                    {
-                        return (AgreementAcceptedDate == null)
-                            ? "User Requested Remote Access"
-                            : "User Approved Remote Access";
-                    }
+                    return (AgreementAcceptedDate == null)
+                        ? "User Requested Remote Access"
+                        : "User Approved Remote Access";
```

Comment:
In the discussions we had with Jolie, there is no difference when the enrollment is accepted or not, they just want to see if the submission had remote access at the time or not. This view model could just return the bool